libs/ts/ui/src/components/Input/Input.tsx

```diff
@@ -32,22 +26,11 @@ const sizes = {
   xl: 'h-14 text-xl px-5',
 };
 
-type InputBaseProps = {
-  className?: string;
+type InputBaseProps = Omit<InputHTMLAttributes<HTMLInputElement>, 'size'> & {
```

Comment:
```suggestion
type InputBaseProps = InputHTMLAttributes<HTMLInputElement & {
```

You can remove `Omit` and rename `size` prop to `inputSize`